### PR TITLE
QA fixes - including FHIR-46035 & FHIR-45390

### PR DESCRIPTION
**Pull request by dbojicic**
This PR addresses the following technical corrections: 
- https://jira.hl7.org/browse/FHIR-46035
  - included dependency to current versions of IPA & Smart App Launch. The version will need to be fixed to a specific version in a publication. Noting there is a qa warning about multiple different potential matches for Smart App Launch (2.2.0 & 2.0.0) due to the IG declaring dependency on the latest and the dependency on the 2.0.0 coming through the IPA.  
- https://jira.hl7.org/browse/FHIR-45390
- and other changes
  - removed dependency on the extension pack 5.1.0-snapshot
  - annotated errors confirmed to be caused by tooling
- fix for https://jira.hl7.org/browse/FHIR-46032

### CODE REVIEW

File: input/ignoreWarnings.txt
Comment (dt-r) [inline input/ignoreWarnings.txt:10]: I'm not sure what the point of adding in the errors to this file? Errors are not ignored.

**Review by dt-r — CHANGES_REQUESTED** [review]
One change requested.

File: input/ignoreWarnings.txt
Comment (dbojicic) [reply input/ignoreWarnings.txt:10]: Errors are not ignored. Build errors related to an artefact (e.g. ActorDefitinion errors) get annotated. Broken links and other errors are still shown including the reason included in the ignoreWarnings but you do need to scroll down to see them. I still see the value in this, and it is what other IGs do, but I do think this was premature and, if we are to do this for other messages, it should be done after the TDG acceptance of the exceptions.

**Comment by dbojicic** [comment]
> One change requested.

Fixed, + an additional Change log update.

File: input/pagecontent/changes.md
Comment (dt-r) [inline input/pagecontent/changes.md:145]: Name the resource the parameter applies to in both instances.

File: input/ignoreWarnings.txt
Comment (dt-r) [reply input/ignoreWarnings.txt:10]: No requirement to wait for TDG approval of an exception to simply annotate the errors.  If the change to this file does that then good. 

I've seen the annotation on US Core, but did not see that equivalent annotation on the AU Core branch and thus could detect no difference to the QA report besides it declaring "22 suppressed errors".

**Review by dt-r — CHANGES_REQUESTED** [review]
1. change requested to changes.md
2. you can annotate errors in ignoreWarnings - your choice

**Comment by dbojicic** [comment]
1. Applied fixes to changes.md
2. Removed obsolete entry: "This element does not match any known slice defined in the profile [http://hl7.org/fhir/StructureDefinition/capabilitystatement-search-parameter-combination|4.0.1](http://hl7.org/fhir/StructureDefinition/capabilitystatement-search-parameter-combination%7C4.0.1)" from ignoreWarning file.  as it is not coming through anymore, or to be precise it's coming through as This element does not match any known slice defined in the profile [http://hl7.org/fhir/StructureDefinition/capabilitystatement-search-parameter-combination|5.1.0](http://hl7.org/fhir/StructureDefinition/capabilitystatement-search-parameter-combination%7C5.1.0) (this may not be a problem, but you should check that it's not intended to match a slice) information message.
3. I haven't annotated errors this time - will do that in one of the next QA PRs.

**Review by dt-r — APPROVED** [review]

**Pull request merged**
